src/lib/constants/index.ts

```diff
@@ -4,6 +4,7 @@ export const chunkedUploaderFileSize = 50_000_000;
 export const bundlrBalanceDecimals = 8;
 export const uploadPriceDecimals = 8;
 export const minimumResurrection = 30_000;
+export const monthSeconds = 2628288;
```

Comment:
This will conflict with the value I'm using for the number of seconds in a month. We should make sure they are the same.

Reply:
@knoll3 -- this is the value being used in the archaeologist service -- what value are you using? We should come to consensus and have it be the same.

The value was taken from this google answer, although # of seconds in 365 days / 12 is `2628000`, which is 288 seconds less (somewhat negligible).

![image](https://user-images.githubusercontent.com/43755993/220766760-e38d77ec-68f8-43a9-87bb-0deecbb7a375.png)